components/form-fields/help-types.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
`pages/add.jsx` and `pages/profile-edit.jsx` both have this field in the form. I extracted the code so this lives on its own file.

Reply:
Nice